src/cp/Cargo.toml

```diff
@@ -8,10 +8,12 @@ name = "uu_cp"
 path = "cp.rs"
 
 [dependencies]
+walkdir = "1"
+quick-error = "1.1.0"
 getopts = "*"
 libc = "*"
 uucore = { path="../uucore" }
-walkdir = "*"
+clap = "*"
```

Comment:
Can you change this to a specific version?  I think we should probably start specifying versions for all of the utils' dependencies...